commands/update/cluster/command.go

```diff
@@ -130,9 +143,21 @@ func printValidation(cmd *cobra.Command, positionalArgs []string) {
 	os.Exit(1)
 }
 
-// updateCluster updates the cluster.
-// It determines whether it is a v5 or v4 cluster and uses the appropriate mechanism.
 func updateCluster(args Arguments) (*result, error) {
+	if args.Labels == nil || len(args.Labels) == 0 {
+		return updateName(args)
+	}
+
+	if args.Name == "" {
+		return updateLabels(args)
+	}
+
+	return nil, microerror.Mask(errors.NoOpError)
+}
+
+// updateName updates the cluster.
+// It determines whether it is a v5 or v4 cluster and uses the appropriate mechanism.
+func updateName(args Arguments) (*result, error) {
```

Comment:
Maybe consider aligning the comments on `updateName` and `updateLabels`. Is the label behavior v5 only?

Reply:
yeah labels are v5 only. I guess I'll add a check like its present in `updateName` to abort the action for v4 clusters :+1: